src/data_provider/src/packages/packageLinuxParserRpm.cpp

```diff
@@ -0,0 +1,45 @@
+
+#include "packageLinuxDataRetriever.h"
+#include "iberkeleyDbWrapper.h"
+#include "berkeleyRpmDbHelper.h"
+#include "sharedDefs.h"
+#include "packageLinuxParserHelper.h"
+#include "filesystemHelper.h"
+#include "rpmlib.h"
+
+void getRpmInfo(std::function<void(nlohmann::json&)> callback)
+{
+    if (!Utils::existsRegular(RPM_DATABASE))
+    {
+        // We are probably using RPM >= 1.17 – get the packages from librpm.
```

Comment:
librpm ndb support was added in 4.16

Reply:
Indeed, DBD was deprecated and NDB moved to stable in 4.16, fixing this...